frontend/src/components/maps/leaflet/InfoSlideOut/InfoContent.tsx

```diff
@@ -80,6 +82,14 @@ export const InfoContent: React.FC<IInfoContent> = ({
   const isParcel =
     propertyTypeId !== null &&
     [PropertyTypes.PARCEL, PropertyTypes.SUBDIVISION].includes(propertyTypeId);
+
+  const lookupCodes = useCodeLookups();
+  const agencies = lookupCodes.getByType(API.AGENCY_CODE_SET_NAME);
+  const getFullAgencyName = (agencyCode?: string) => {
```

Comment:
we may want this elsewhere, could put this in the useCodeLookups hook.

Reply:
I think the though crossed my mind at one point and then I forgot, I'll do that :)